### PR TITLE
interceptor changes to encode URL

### PR DESCRIPTION
**Pull request by madhavmansuriya40**
reference: https://stackoverflow.com/questions/45428842/angular-url-plus-sign-converting-to-space/52458069#52458069

### CODE REVIEW

**Review by iamtarun1993 — APPROVED** [review]

**Comment by iamtarun1993** [comment]
Add the StackOverflow reference here

File: src/app/core/interceptors/httpInterceptor.ts [hunk 1]
```diff
@@ -78,6 +80,8 @@ export class HttpConfigInterceptor implements HttpInterceptor {
         concatMap(({token, deviceInfo}) => {
           if (token && this.secureUrl(request.url)) {
             request = request.clone({ headers: request.headers.set('Authorization', 'Bearer ' + token) });
+            const params = new HttpParams({encoder: new CustomEncoder(), fromString: request.params.toString()});
```
Comment (Julias0) [inline src/app/core/interceptors/httpInterceptor.ts:83]: What is there inside `request.params`

Comment (madhavmansuriya40) [reply src/app/core/interceptors/httpInterceptor.ts:83]: it will consist of payload for post call

Comment (madhavmansuriya40) [reply src/app/core/interceptors/httpInterceptor.ts:83]: Screenshot 2021-03-02 at 2 00 35 PM

**Review by Julias0 — APPROVED** [review]

**Review by abhishek1234321 — APPROVED** [review]

**Pull request merged**